include/mbedtls/check_config.h

```diff
@@ -688,6 +688,14 @@
 #error "MBEDTLS_HAVE_INT32/MBEDTLS_HAVE_INT64 and MBEDTLS_HAVE_ASM cannot be defined simultaneously"
 #endif /* (MBEDTLS_HAVE_INT32 || MBEDTLS_HAVE_INT64) && MBEDTLS_HAVE_ASM */
 
+#if defined(MBEDTLS_SSL_HW_RECORD_ACCEL)
+#if defined(MBEDTLS_DEPRECATED_REMOVED)
+#error "MBEDTLS_SSL_HW_RECORD_ACCEL is deprecated and will likely be removed in a future version of Mbed TLS"
+#elif defined(MBEDTLS_DEPRECATED_WARNING)
+#warning "MBEDTLS_SSL_HW_RECORD_ACCEL is deprecated and will likely be removed in a future version of Mbed TLS"
```

Comment:
Again, remove 'likely'.

Reply:
This wording is used in multiple other places in the library. @andresag01 Would you mind also changing those?

Reply:
@hanno-arm: Indeed, this wording was used here because I copied from the documentation of previously deprecated features. Do you mean that we should update the documentation for all of those? I would certainly be in favour of this, but there are *a lot* of features that have this wording and I think it would be best to do that in a separate PR.

Reply:
@andresag01 Yes, I meant making the wording uniform across the library. I think it's worth doing it here, but I'm also ok keeping for a separate PR.